src/binary16.rs

```diff
@@ -291,6 +291,43 @@ impl f16 {
         self.0
     }
 
+    /// Converts `self` to little endian from the target's endianness.
+    ///
+    /// On little endian this is a no-op. On big endian the bytes are swapped.
+    #[inline]
+    pub const fn to_le(self) -> u16 {
```

Comment:
I don't think `to_le` and `to_be` make a lot of sense for `f16`; they aren't in `f32` and `f64` either. It would be more helpful to have `from_be_bytes`, `from_le_bytes` and `from_ne_bytes` like `f32` and `f64` do.

Reply:
I'm inclined to agree with tspiteri here. As f32/f64 don't have these methods, they're more for integer types than floats. The same conversion can be done slightly more verbosely with `.to_bits().to_le()`, etc., without the possible confusion with `to_le_bytes`.

I'm gonna merge but then remove `to_le` and `to_be` and add `from_*_bytes` myself.